test/absinthe/type/object_test.exs

```diff
@@ -129,4 +129,19 @@ defmodule Absinthe.Type.ObjectTest do
       end
     end)
   end
+
+  # TODO default_value is always nil????
+  describe "object field default_value evaluation" do
+    Absinthe.Fixtures.FunctionEvaluationHelpers.function_evaluation_test_params()
+    |> Enum.each(fn %{
+                      test_label: test_label,
+                      expected_value: expected_value
+                    } ->
+      test "for #{test_label} (evaluates default_value to '#{expected_value}')" do
+        type = Object.TestSchemaFieldsAndArgsDescription.__absinthe_type__(:field_default_value)
+
+        assert type.fields[unquote(test_label)].default_value == unquote(expected_value)
+      end
+    end)
```

Comment:
the input object's field's default_value is always nil here. Not sure if intentional or not. But i have removed this test